src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/AsyncMethodBuilder.cs

```diff
@@ -29,24 +29,24 @@ public struct AsyncVoidMethodBuilder
     {
         /// <summary>The synchronization context associated with this operation.</summary>
         private SynchronizationContext? _synchronizationContext;
-        /// <summary>The builder this void builder wraps.</summary>
-        private AsyncTaskMethodBuilder _builder; // mutable struct: must not be readonly
+        /// <summary>The lazily-initialized built task.</summary>
+        private Task<VoidTaskResult> m_task; // lazily-initialized: must not be readonly. Debugger depends on the exact name of this field.
```

Comment:
The "must not be readonly" comment can be removed if it's no longer a mutable struct.